orunmila.go

```diff
@@ -230,7 +230,7 @@ func searchWordsByTagIds(db *sql.DB, tags string) {
 		log.Infoln("Using tags:", Tags)
 		queryStr = fmt.Sprintf(queryStr+" left join wt as t2 on t2.word_id=t1.id WHERE t2.tag_id IN (%s) group by t1.id", TagsToString())
 	} else {
-		log.Infoln("No tags were given")
+		log.Infoln("No tags were given or tags no found")
```

Comment:
No if tags were given and not found it should return nothing... The idea is the when the user do not provide -tags we assume that they want to list all entries in the database, but if they provide eg `-tags nonexistant_tag` we dont want to return the entire database.